### PR TITLE
[outdated] Add post #13

### PR DESCRIPTION
**Pull request by batareechka**
"Improving User Experience for Multilingual Web Browsing" – an article about using `translate` attribute.

Feedback is very much welcome 🙌🏻

### CODE REVIEW

File: hell/adventcalendar/2024/13/index.md [hunk 1]
```diff
@@ -1,52 +1,68 @@
...
-* Lorem, ipsum.
+```html
+<!-- Original content in English -->
+<p>Name: <span translate="no">Joke</span></p>
+<p>Email: <span translate="no">joke@test.com</span></p>
+
+<!-- Translated content in Dutch -->
+<p>Naam: <span translate="no">Joke</span></p>
+<p>E-mailadres: <span translate="no">joke@test.com</span></p>
+```
```
Comment (batareechka) [inline hell/adventcalendar/2024/13/index.md:55]: I was thinking about giving even more examples and maybe going deeper into which elements are not auto-translated...

For now, I have focused on a "snippet" idea and making it short. Let me know if additional info and context would be helpful!

**Comment by matuzo** [comment]
Hey Anastasiia,
thanks a lot for your great post. We've added some comments directly in your file in the advent24 branch of the HTMHell repository. Sorry that we're not using GitHub and it's comment feature as intended but our approach makes it easier for us since not all authors are using GitHub.

Please review our comments, make your changes if necessary, and submit another PR. If you have questions please ask them here.

Thanks a lot,
Manuel, Saptak, and Karl

**Comment by batareechka** [comment]
Hey @matuzo,

Thank you for the feedback! I'm on it, will push changes later this week

**Comment by batareechka** [comment]
Hi @matuzo, apologies for the delay.
I have created [another PR](https://github.com/matuzo/HTMHell/pull/176) with fixes. Please have a look.
This PR I'll close as soon as all work on the article is done.

**Pull request closed without merging**